boards/arm/mps2_an385/mps2_an385_defconfig

```diff
@@ -28,3 +28,11 @@ CONFIG_UART_CMSDK_APB=y
 CONFIG_WATCHDOG=y
 
 CONFIG_I2C=y
+
+#Enable MPU
+CONFIG_ARCH_HAS_USERSPACE=y
```

Comment:
that is not needed and is automatically selected.

Reply:
This was left over code that came from the qemu enabling patches. Removed it.